client/src/js/view/components/settings/settings-about.js

```diff
@@ -1,24 +1,31 @@
 import { h } from 'hyperapp';
+import { Link } from '../elements/link';
 
 export const SettingsAbout = ({ state }) =>
-    <div class="settings_content">
-        <div>Turtle Control Software v{state.system_info.tcs_ver}</div>
-        <div>Shield Firmware v{state.system_info.firmware_ver}</div>
-        <div>External WiFi: {state.system_info.wifi_dongle}</div>
-        <div>
-            {
-                state.system_info.video_devices.map(device => (<VideoDevice device={device} />))
-            }
+    <div class='settings__content'>
+        <div class='settings__block'>
+            <div class='settings__subtitle'>system info</div>
+            <div class="settings__section">Turtle Control Software v{state.system_info.tcs_ver}</div>
+            <div class="settings__section">Shield Firmware v{state.system_info.firmware_ver}</div>
+            <div class="settings__section">External WiFi: {state.system_info.wifi_dongle}</div>
+            <div class="settings__section">{state.system_info.video_devices.map(device => (<VideoDevice device={device} />))}</div>
+        </div>
+        <div class='settings__block'>
+            <div class='settings__subtitle'>Documentation</div>
+            <div class="settings__section">
+                Please refer to &nbsp;<Link href='https://docs.turtlerover.com/' text='docs.turtlerover.com' /> &nbsp;to know more about the control options and the Rover maintenance.
+            </div>
+        </div>
+        <div class='settings__block'>
+            <div class='settings__subtitle'>A word from a team</div>
+            <div class="settings__section">
+                Feel free to contact us if you have any issue. We're constantly improving the UI, but it's still a long way to go.<br />
```

Comment:
@SzymonDzw Check text in about

Reply:
Should be "a word from **the** team". Rest is ok.

Reply:
"our" might fit even better.